cetav-ddw/lalibertad-cetav
# FAQ component

## Description

I have completed the task, which involved creating a reusable component that displays the title "¿Tiene Preguntas para Nosotros?" and a text that says "Resuélvelas en nuestra sección de Preguntas Frecuentes." The text is a link that will direct users to the Frequently Asked Questions page.

### How to verify this PR

To verify the changes made in this pull request, follow these steps:

1. Navigate to the relevant page where the component is intended to be displayed.
2. Verify that the title "¿Tiene Preguntas para Nosotros?" is present.
3. Check if the text "Resuélvelas en nuestra sección de Preguntas Frecuentes" is displayed below the title.
4. Click on the link to ensure that it correctly directs users to the Frequently Asked Questions page.
5. Ensure that this component has been added to the appropriate pages where it is intended to be used.

![Captura de Pantalla 2023-10-30 a la(s) 21 01 01](https://github.com/cetav-ddw/lalibertad-cetav/assets/84554799/2730c1da-283b-4bc6-9040-34305a0bacdb)
